libraries/AP_Scripting/drivers/LTE_modem.lua

```diff
@@ -211,11 +211,11 @@ local LTE_TX_RATE  = bind_add_param('TX_RATE',  20, 0)
 --[[
     // @Param: LTE_BAND
     // @DisplayName: LTE band selection
-    // @Description: This allows selection of LTE band, 0 for no preference
-    // @Range: 0 50
+    // @Description: This allows selection of LTE band, 0 for no preference, -1 for no change
```

Comment:
I think this could be a little clearer, perhaps -1: do not set band 0: set all band otherwise the specific band required.

Reply:
changed